__tests__/app.test.js

```diff
@@ -71,8 +71,70 @@ describe("GET /api/reviews", () => {
         });
       });
   });
-});
+  test("only returns reviews of a given category when queried by category", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .then(({ body }) => {
+        const { reviews } = body;
+        reviews.forEach((review) => {
+          expect(review.category).toBe("dexterity");
+        });
+      });
+  });
+  it("should return an empty array if the given category query has no reviews", () => {
+    return request(app)
+      .get("/api/reviews?category=test")
```

Comment:
I think this should be a 404 as this is a non-existent category rather than one with no reviews